internal/states/statefile/version4.go

```diff
@@ -764,6 +767,8 @@ type outputStateV4 struct {
 	ValueRaw     json.RawMessage `json:"value"`
 	ValueTypeRaw json.RawMessage `json:"type"`
 	Sensitive    bool            `json:"sensitive,omitempty"`
+	// TODO: Do we need to update the state schema or state documentation somewhere?
```

Comment:
We typically don't suggest folks look directly at the statefile, I don't think we have these fields documented anywhere concrete.